src/main/java/uk/gov/cshr/civilservant/domain/Question.java

```diff
@@ -28,15 +34,37 @@
     @Column(length = 500)
     private String learningReference;
 
-    @Column(length = 500)
+    @Column(length = 500, unique = true)
```

Comment:
Does a question need to be unique? Because I think it's possible to have 2 quizzes of the same profession with one of them being inactive to have the same question

Reply:
oops. this should have been removed. my bad.we need to introduce a unique constraint in the backend where one quiz cannot have 2 similar questions.

Reply:
based on our design, questions become inactive when quiz is archived.